src/Phpmig/Adapter/Illuminate/Database.php

```diff
@@ -21,13 +21,13 @@ class Database implements AdapterInterface
     protected $tableName;
 
     /**
-     * @var \Illuminate\Database\Capsule\Manager
+     * @var \Illuminate\Database\Connection
```

Comment:
Ah, my bad, read the diff incorrectly, I thought I was looking at the docblock for the constructor.